components/bl-mapbox/component.json

```diff
@@ -204,6 +204,10 @@
         {
           "id": "description",
           "label": "Description"
+        },
+        {
+          "id": "markerData",
```

Comment:
Seems strange when Marker entity conteint `markerData` property. That should be `data` or something that means additional information inside market